reddit_decider/__init__.py

```diff
@@ -352,32 +339,28 @@ def get_variant(
 
         :return: Variant name if a variant is assigned, :code:`None` otherwise.
         """
-        decider = self._get_decider()
-        if decider is None:
+        if self._rs_decider is None:
+            logger.error("rs_decider is None--did not initialize.")
             return None
```

Comment:
Instead of checking that the decider is initialized on every `choose()`, can we check that the value passed in to the constructor is not `None`?

Reply:
after discussion, leaving this as-is since that's current behavior.